samples/sensor/thermometer/sample.yaml

```diff
@@ -18,3 +18,18 @@ tests:
         - "Thermometer Example (.*)"
         - "Temperature device is 0x[0-9|a-z]+, name is [a-z|0-9]+@[a-z|0-9]+"
         - "Temperature is [0-9|.]+°C"
+  sample.sensor.thermometer.p3t1755dp_ard:
+    tags: sensors
+    filter: dt_alias_exists("ambient-temp0")
+    extra_args: SHIELD=p3t1755dp_ard
+    platform_allow:
+      - mimxrt1180_evk/mimxrt1189/cm33   # p3t1755
```

Comment:
do not add this

Reply:
Thank you for pointing out the issue. I have updated in yml.